src/particle/corsika/gen_conversion_table.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2018-2022, Eduardo Rodrigues and Henry Schreiner.
```

Comment:
Same as for the LHCb script, can you move this script to the admin/ directory and make the script name a bit more specific?